File: NodeJS/State/Rojo.js
const {IEstado} = require("./IEstado");

class Rojo extends IEstado{

    retirarSaldo(saldoARet){

        if(saldoARet > 1000){
            console.log("Puede retirar $1000 como maximo");
            saldoARet = 1000;
        }

        console.log("Retira $" + saldoARet);

    }

}

exports.Rojo = Rojo;

File: Python/Rojo.py
from IEstado import IEstado

class Rojo(IEstado):

    def retirarSaldo(self, saldoARet: float):

        if(saldoARet > 1000):
            print("Puede retirar $1000 como maximo")
            saldoARet = 1000

        print("Retira $ {}".format(saldoARet))